packages/ui-components/src/utils/date.helper.ts

```diff
@@ -10,6 +10,7 @@ import isBetween from 'dayjs/plugin/isBetween';
 import quarterOfYear from 'dayjs/plugin/quarterOfYear';
 import utc from 'dayjs/plugin/utc';
 import { TIMEZONES } from './date.config';
+import { SelectedRange } from '../components/calendar/calendar.types';
```

Comment:
should this SelectedRange be defined in more high level of the apps, seems usage in a lot of places to be categorized on calendar.types

Reply:
Yep, I agree. Some of the types can be in a higher level. I will move this one but I am expecting to move a lot more and have a better understanding about the types location after I start messing with the time picker components.